register/tests.py

```diff
@@ -238,3 +241,30 @@ def test_invalid_register_student(self):
         response = self.client.post(url, data=data)
         self.assertEqual(response.status_code, 200)
         self.assertEqual(Student.objects.filter(username="hritik").count(), 0)
+
+
+class StudentViewActivateTest(TestCase):
+    def create_student(self):
+        return Student.objects.create(
+            first_name="Hritik",
+            last_name="Roshan",
+            email_address="hrx@gmail.com",
+            phoneNumber="9567801234",
+            username="hritik",
+            password="hritikRoshan@10",
+            current_school="NYU",
+            borough="MN",
+        )
+
+    def setUp(self):
+        self.student = self.create_student()
+
+    def test_activation_student(self):

```

Comment:
need to also write a test case where this fails (if possible)

Reply:
done